draft-ietf-quic-qpack.md

```diff
@@ -937,11 +939,11 @@ entries, the Base will be greater than the Required Insert Count, so the delta
 will be positive and the sign bit is set to 0.
 
 An encoder that produces table updates before encoding a header block might set
-Required Insert Count and the Base to the same value.  In such case, both the
-sign bit and the Delta Base will be set to zero.
+Base to the value of Required Insert Count.  In such case, both the sign bit and
+the Delta Base will be set to zero.
 
 A header block that does not reference the dynamic table can use any value for
-the Base; setting Delta Base to zero is the most efficient encoding.
+the Base; setting Delta Base to zero is one of the most efficient encodings.
```

Comment:
It's true there are other 2 byte encodings, but I think we should encourage 0/0.  Maybe we should even require it?